sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Snippets/Sample01_HelloWorldLiveTests.cs

```diff
@@ -0,0 +1,277 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Producer;
+using Azure.Messaging.EventHubs.Processor;
+using Azure.Messaging.EventHubs.Processor.Tests;
+using Azure.Storage.Blobs;
+using NUnit.Framework;
+using System.Diagnostics;
+using System.Collections.Concurrent;
+
+namespace Azure.Messaging.EventHubs.Tests.Snippets
+{
+    /// <summary>
+    ///   The suite of tests defining the snippets used in the Event Hubs
+    ///   Sample01_HelloWorld sample.
+    /// </summary>
+    ///
+    [TestFixture]
+    [Category(TestCategory.Live)]
+    [Category(TestCategory.DisallowVisualStudioLiveUnitTesting)]
+    [SuppressMessage("Style", "IDE0059:Unnecessary assignment of a value", Justification = "Example assignments needed for snippet output content.")]
+    public class Sample01_HelloWorldLiveTests
```

Comment:
Consider using SampleBase<T>: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core.TestFramework/src/SamplesBase.cs

Reply:
Thanks for the heads-up.   I wasn't aware of it, nor LiveTestBase.   I'll take a note to think about moving the tests across both Event Hubs libraries over as a follow-up, but I'll defer for now.